docs/source/projections/cass.rst

```diff
@@ -1,10 +1,142 @@
 .. _cass:
 
 ********************************************************************************
-Cassini
+Cassini (Cassini-Soldner)
 ********************************************************************************
 
+Although the Cassini projection has been largely replaced by the Transverse Mercator, it is still in limited use outside the United States and was one of the major topographic mapping projections until the early 20th century.
+
++---------------------+-------------------------------------------------------------------------+
+| **Classification**  | Transverse and oblique cylindrical                                      |
++---------------------+-------------------------------------------------------------------------+
+| **Available forms** | Forward and inverse, Spherical and Elliptical                           |
++---------------------+-------------------------------------------------------------------------+
+| **Defined area**    | Global, but best used near the central meridian with long, narrow areas |
++---------------------+-------------------------------------------------------------------------+
+| **Implemented by**  | Gerald I. Evenden                                                       |
++---------------------+-------------------------------------------------------------------------+
+| **Options**                                                                                   |
++---------------------+-------------------------------------------------------------------------+
+| `+lat_0`            | Center of the map : latitude of origin                                  |
++---------------------+-------------------------------------------------------------------------+
+
 .. image:: ./images/cass.png
    :scale: 50%
    :alt:   Cassini  
 
+Usage
+########
+
+There has been little usage of the spherical version of the Cassini, but the ellipsoidal Cassini-Soldner version was adopted by the Ordnance Survey for the official survey of Great Britain during the second half of the 19th century (Steers, 1970, p. 229). Many of these maps were prepared at a  scale of 1:2,500. The Cassini-Soldner was also used for the detailed mapping of many German states during the same period. 
```

Comment:
Thanks! It would be cool to also have the references listed on each page, but I am not sure sphinx can do that. At least not without cluttering other types of output like pdf.